specs/wp-signup-spec.js

```diff
@@ -1399,4 +1400,87 @@ test.describe( `[${ host }] Sign Up  (${ screenSize }, ${ locale })`, function()
 			} );
 		}
 	);
+
+	test.describe( 'Sign up for free subdomain site @parallel', function() {
+		this.bailSuite( true );
+
+		const blogName = dataHelper.getNewBlogName();
+		const expectedDomainName = `${ blogName }.art.blog`;
+
+		test.it( 'Ensure we are not logged in as anyone', async function() {
+			return await driverManager.ensureNotLoggedIn( driver );
+		} );
+
+		test.it( 'Can enter the subdomains flow and select design type', async function() {
+			await driver.get( config.get( 'calypsoBaseURL' ) + '/start/subdomain/?vertical=a8c.1' );
+			const designTypePage = await DesignTypePage.Expect( driver );
+			await designTypePage.setABTestControlGroupsInLocalStorage( { flow: 'subdomain' } );
```

Comment:
You can call this off StartPage like the free scenario since this refreshes the page and can cause problems with the page changing based upon AB test groups